contracts/Gate.sol

```diff
@@ -95,7 +95,7 @@ contract Gate is ProtocolVersioned, Owned {
         uint256 _amount,
         address _beneficiary)
         external
-        returns ( bool /* success */)
+        returns (bool isSuccess)
```

Comment:
please don't declare `isSuccess` as a variable if unused;
prefer `return (bool /* success */)`

Reply:
OK, It was added to make all the method signatures look consistent, also it will tell what is meaning of bool return type(here it tells if the operation is successful)